sheeprl/algos/ppo_pixel/ppo_pixel_continuous.py

```diff
@@ -44,17 +44,22 @@ def make_env(
     vector_env_idx: int = 0,
     action_repeat: int = 2,
     screen_size: int = 64,
+    frame_stack: int = 0,
 ):
     def thunk():
         env = gym.make(env_id, render_mode="rgb_array")
         env = ActionRepeat(env, action_repeat)
-        if isinstance(env.observation_space, gym.spaces.Box) or len(env.observation_space.shape) < 3:
```

Comment:
I would add in the help string of the args (or in the readme or somewhere else) that the observation space of the environment must be compatible with the [`PixelObservationWrapper`](https://gymnasium.farama.org/api/wrappers/observation_wrappers/#gymnasium.wrappers.PixelObservationWrapper), otherwise it raises an exception. What do you think?

Reply:
What are the requirements? Cannot we just check it at runtime?

Reply:
The documentation states that: "supported formats are a single array, or a dict of arrays".
Yes, we can check it runtime. I thought it might be useful to warn the user to choose the environment properly, to avoid wasting too much time behind runtime errors.